packages/bpk-mixins/src/mixins/_badges-v2.scss

```diff
@@ -0,0 +1,236 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../bonds.scss';
+@import 'typography.scss';
+
+////
+/// @group badges
+////
+
+/// Standard badge.
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge-v2();
+///   }
+
+@mixin bpk-badge-v2 {
+  display: inline-flex;
+  padding: bpk-spacing-sm() bpk-spacing-md();
+  align-items: center;
+
+  @include bpk-border-radius-xs;
+  @include bpk-text;
+  @include bpk-footnote;
+}
+
+/// Centered badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge-v2();
+///     @include bpk-badge-v2--centered();
+///   }
+
+@mixin bpk-badge-v2--centered {
+  vertical-align: text-bottom;
+}
+
+/// Right-docked badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge-v2();
+///     @include bpk-badge-v2--docked-right();
+///   }
+
+@mixin bpk-badge-v2--docked-right {
+  position: absolute;
+  top: 0;
+  right: 0;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+
+  @include bpk-rtl {
+    right: inherit;
+    left: 0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: $bpk-border-radius-xs;
+  }
+}
+
+/// Left-docked badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge-v2();
+///     @include bpk-badge-v2--docked-left();
+///   }
+
+@mixin bpk-badge-v2--docked-left {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  border-bottom-left-radius: 0;
+
+  @include bpk-rtl {
+    right: 0;
+    left: inherit;
+    border-bottom-left-radius: $bpk-border-radius-xs;
+    border-bottom-right-radius: 0;
+  }
+}
+
+/// Normal badge. Modifies the bpk-badge mixin.
+///
+/// @require {mixin} bpk-badge
+///
+/// @example scss
+///   .selector {
+///     @include bpk-badge-v2();
+///     @include bpk-badge-v2--normal();
+///   }
+
+@mixin bpk-badge-v2--normal {
+  background-color: $bpk-canvas-contrast-day;
```

Comment:
Figma says new token to be made, can't see where and this colour already exists - let me know if there's somewhere else I should be adding this.

![image](https://github.com/Skyscanner/backpack/assets/22656572/8d095adf-864f-41f0-b9b5-fbcb5a4f8383)

Reply:
Hello 👋 so our colours exist and any to be made here in our Backpack Foundations repositiory https://github.com/Skyscanner/backpack-foundations/

Reply:
Thanks! I've now pulled in the new version and using the variable 🎉